charts/gateway-addons-helm/README.md

```diff
@@ -0,0 +1,86 @@
+# gateway-addons-helm
```

Comment:
Are the changes in gateway-addons-helm intentional?

Reply:
fixed via https://github.com/envoyproxy/gateway/pull/3555